packages/rpc-core/src/rpc-methods/__tests__/get-minimum-balance-for-rent-exemption-test.ts

```diff
@@ -0,0 +1,28 @@
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+
+import { Commitment } from '../common';
+import { createSolanaRpcApi, SolanaRpcMethods } from '../index';
+
+describe('getMinimumBalanceForRentExemption', () => {
+    let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        rpc = createJsonRpc<SolanaRpcMethods>({
+            api: createSolanaRpcApi(),
+            transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        });
+    });
+
+    (['confirmed', 'finalized', 'processed'] as Commitment[]).forEach(commitment => {
+        describe(`when called with \`${commitment}\` commitment`, () => {
+            it('returns an expected rent amount', async () => {
+                expect.assertions(1);
+                const result = await rpc.getMinimumBalanceForRentExemption(BigInt(0)).send();
```

Comment:
```suggestion
                const result = await rpc.getMinimumBalanceForRentExemption(BigInt(0), { commitment }).send();
```

Reply:
@2501babe If you want to commit this change we can merge this in!

Reply:
ah my mistake, thank you!